CMakeLists.txt

```diff
@@ -227,6 +227,8 @@ target_link_libraries(${PROJECT_NAME} ${_INTERFACE_OR_PUBLIC}
 
 # Set the definitions to enable optional features
 target_compile_definitions(${PROJECT_NAME} ${_INTERFACE_OR_PUBLIC}
+	# Require Windows 8
+	$<$<PLATFORM_ID:Windows>:_WIN32_WINNT=0x0602>
```

Comment:
@sum01 @jimmy-park could you review the change in CMakeLists.txt, since I am not an expert of CMake? Thanks a lot.

Reply:
Won't this affect other CMake projects that use httplib?
I think the macro should remain in `httplib.h`.

Reply:
@jimmy-park thank you for the comment.
@sergio-nsk could you remove this change?

Reply:
> Won't this affect other CMake projects that use httplib? I think the macro should remain in `httplib.h`.

It will affect user projects if `HTTPLIB_COMPILE` is `ON`.